src/containers/AccountConnection.jsx

```diff
@@ -187,28 +202,48 @@ class AccountConnection extends Component {
         </div>
         <div className={styles['col-account-connection-content']}>
           <div className={styles['col-account-connection-form']}>
-            { account
-              ? <h4>{t('account.connection.account.title')}</h4>
-              : <div>
-                <h3>{t('account.connection.title', { name: connector.name })}</h3>
-                <p className={styles['col-account-connection-security']}>
-                  <svg>
-                    <use xlinkHref={securityIcon} />
-                  </svg>
-                  {t('account.connection.security')}
+            { error
+              ? <div className='coz-error'>
+                <h4>{t('account.connection.error.title')}</h4>
+                <p>
+                  <ReactMarkdown
+                    source={t('account.connection.error.description', {name: connector.name, forum: t('account.connection.error.forum')})}
+                    renderers={{Link: props => <a href={props.href} target='_blank'>{props.children}</a>}}
+                  />
                 </p>
               </div>
+
+              : existingAccount
+                ? <h4>{t('account.connection.account.title')}</h4>
+                : <div>
+                  <h3>{t('account.connection.title', { name: connector.name })}</h3>
+                  <p>{t('account.connection.description', { name: connector.name })}</p>
+                  <p>
+                    <ReactMarkdown
+                      source={
+                        t(description)
+                      }
+                      renderers={{Link: props => <a href={props.href} target='_blank'>{props.children}</a>}}
+                    />
+                  </p>
+                  <p className={styles['col-account-connection-security']}>
+                    <svg>
+                      <use xlinkHref={securityIcon} />
+                    </svg>
+                    {t('account.connection.security')}
+                  </p>
+                </div>
             }
             <AccountLoginForm
               t={t}
               konnector={connector}
               customView={customView}
               fields={fields}
-              dirty={dirty}
               submitting={submitting}
               deleting={deleting}
-              values={account ? account.auth : {}}
+              values={existingAccount ? existingAccount.auth : {}}
```

Comment:
@CPatchane  You're right, I need to test. How do you suggest to pass it ? Something like `values={existingAccount ? existingAccount.auth || existingAccount.oauth : {}}` ?